pywbem/config.py

```diff
@@ -93,3 +93,17 @@
 #:
 #: *New in pywbem 0.12.*
 SEND_VALUE_NULL = True
+
+#: Backward compatibility option that controls creating CIMInstanceNames with
+#: null values for keybindings.
+
+#: :term:`DSP0004`, section 7.9 specifically forbids key properties with
+#: no values but because pywbem has always allowed this, adding the code to
```

Comment:
Instead of saying "with no values", I suggest to say "that are NULL".

Reply:
DONE